lib/cli/placeorder.ts

```diff
@@ -62,6 +64,11 @@ export const placeOrderHandler = (argv: Arguments<any>, side: OrderSide) => {
     return;
   }
 
+  if (checkDecimalPlaces(numericPrice)) {
+    console.log('price cannot have more than 12 decimal places');
```

Comment:
We should log this to `console.error` and also set the status code to a non-zero value to indicate the call has failed with `process.exitCode = 1`.

Reply:
Done